std.ddoc

```diff
@@ -37,3 +37,6 @@ JOIN_DOT_TAIL=$(JOIN_DOT $+)
 JOIN_DOT=.$1$(JOIN_DOT $+)
 TAIL=$+
 _=
+
+DDOC_BLANKLINE = $(BR)$(BR)$(LF)
```

Comment:
The extra LF in there is for compatibility with the existing generated HTML. We may eliminate it later if we find it fit.